web/src/main/java/org/cbioportal/weblegacy/CNASegmentController.java

```diff
@@ -0,0 +1,29 @@
+/**
+ *
+ * @author jiaojiao
+ */
+
+package org.cbioportal.weblegacy;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.List;
+import org.cbioportal.model.CNASegmentData;
+import org.cbioportal.service.CNASegmentService;
+
+@RestController
+public class CNASegmentController {
+
+    @Autowired
+    private CNASegmentService cnaSegmentService;
+ 
```

Comment:
Add some human readable descriptions with @ApiOperation and @ApiParam annotations -- see GenePanelController.java for examples